scripts/badger/thresher.py

```diff
@@ -0,0 +1,172 @@
+    CVXCRV.approve(BCVXCRV, 0)
+    CVX.approve(BVECVX, CVX.balanceOf(SAFE))
+    BVECVX.depositAll()
+
+
+def unwind_lps():
+    # build list with lp token labels
+    lps = []
+    for lp in registry.eth.treasury_tokens.keys():
+        if lp.startswith('uni') or lp.startswith('slp') or lp.startswith('crv'):
+            lps.append(lp)
+
+    # withdraw from uni and sushi lps
+    for label in lps:
+        lp = Contract(registry.eth.treasury_tokens[label], owner=SAFE.account)
+        bal_start = lp.balanceOf(SAFE)
+        if bal_start <= 0:
+            continue
+        if label.startswith('uni'):
+            SAFE.uni_v2.remove_liquidity(lp, bal_start)
+        if label.startswith('slp'):
+            SAFE.sushi.remove_liquidity(lp, bal_start)
+
+    # since some btokens hold curve lps we withdrawall btokens, except for
+    # productive ones
+    setts = registry.eth.sett_vaults.copy()
+    setts.pop('bcrvIbBTC')
+    setts.pop('bcvxCRV')
+    setts.pop('bveCVX')
+    setts.pop('bbveCVX-CVX-f')
+    for addr in setts.values():
+        sett = interface.ISettV4h(addr, owner=SAFE.account)
+        if sett.balanceOf(SAFE) > 0:
+            sett.withdrawAll()
+
+    # withdraw from curve lps
+    for label in lps:
+        if label == 'crv3pool':
+            continue
+        lp = Contract(registry.eth.treasury_tokens[label], owner=SAFE.account)
+        bal_start = lp.balanceOf(SAFE)
+        if bal_start <= 0:
+            continue
+        if label.startswith('crv'):
+            try:
+                SAFE.curve.withdraw_to_one_coin(lp, bal_start, SBTC_LP)
+            except:
+                try:
+                    SAFE.curve.withdraw_to_one_coin(lp, bal_start, WBTC)
+                except:
+                    try:
+                        SAFE.curve.withdraw_to_one_coin(lp, bal_start, THREEPOOL)
+                    except:
+                        SAFE.curve.withdraw(lp, bal_start)
+
+
+def dogfood_btc():
+    # zap ibbtc, renbtc, wbtc and sbtc into the ibbtc_lp
+    dep_ibbtc = IBBTC.balanceOf(SAFE) * DUSTY
+    dep_renbtc = RENBTC.balanceOf(SAFE) * DUSTY
+    dep_wbtc = WBTC.balanceOf(SAFE) * DUSTY
+    dep_sbtc = SBTC.balanceOf(SAFE) * DUSTY
+    expected = ZAP.calc_token_amount(
+        IBBTC_LP,
+        [dep_ibbtc, dep_renbtc, dep_wbtc, dep_sbtc],
+        True
+    )
+    if dep_ibbtc > 0:
+        IBBTC.approve(ZAP, dep_ibbtc)
+    if dep_renbtc > 0:
+        RENBTC.approve(ZAP, dep_renbtc)
+    if dep_wbtc > 0:
+        WBTC.approve(ZAP, dep_wbtc)
+    if dep_sbtc > 0:
+        SBTC.approve(ZAP, dep_sbtc)
+    ZAP.add_liquidity(
+        IBBTC_LP,
+        [dep_ibbtc, dep_renbtc, dep_wbtc, dep_sbtc],
+        expected * SLIPPAGE
+    )
+
+    # deposit sbtc_lp directly into ibbtc_lp
+    SAFE.curve.deposit(IBBTC_LP, [0, SBTC_LP.balanceOf(SAFE) * DUSTY])
+
+    # finally dogfood all into our own ibbtc sett
+    IBBTC_LP.approve(BIBBTC, 2**256-1)
+    BIBBTC.depositAll()
+    IBBTC_LP.approve(BIBBTC, 0)
```

Comment:
I think this section could be done less dusty relying into this [zapper](https://github.com/Badger-Finance/badger-multisig/blob/main/helpers/addresses.py#L436) instead

The following [deposit](https://etherscan.io/address/0xc5b45ab3d237b0dfc8f8808df39d49ebde8171fe#code#L1364)  method would be handy

Reply:
looking into at it more, i think it will leave hanging the sbtc_lp balance if going the other route...